test/tools/auth0/handlers/connections.tests.js

```diff
@@ -1,3 +1,4 @@
+/* eslint-disable consistent-return */
```

Comment:
Just having some issue with the strictness of the linter, couldn't find a easier way to refactor.